ElectronNET.WebApp/ElectronNET.WebApp.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
     <RuntimeIdentifiers>win-x64</RuntimeIdentifiers>
```

Comment:
not part of this PR but do we really need the RuntimeIdentifiers here?  I've run the app in macos just fine as well..